brainiak/reprsimil/brsa.py

```diff
@@ -1291,16 +1297,22 @@ def _loglike_AR1_diagV_fitU(self, param, XTX, XTDX, XTFX, YTY_diag,
                                  L, rho1, n_V, n_base)
 
         # Only starting from this point, SNR2 is involved
-        LL, LAMBDA_i, LAMBDA, YTAcorrXL_LAMBDA, sigma2 \
-            = self._calc_LL(rho1, LTXTAcorrXL, LTXTAcorrY, YTAcorrY, X0TAX0,
-                            SNR2, n_V, n_T, n_run, rank, n_base)
+
+        try:
+            LL, LAMBDA_i, LAMBDA, YTAcorrXL_LAMBDA, sigma2 \
+                = self._calc_LL(rho1, LTXTAcorrXL, LTXTAcorrY, YTAcorrY,
+                                X0TAX0, SNR2, n_V, n_T, n_run, rank, n_base)
+        except:
```

Comment:
You must specify a particular exception and you must add that exception to the docstring of the function in a section called "Raises".